### ISSUE TITLE
Book Building limit

### ISSUE THREAD

**Issue by SergejSoft** [Neufund/platform-frontend#1609]
We need to implement limit for people taking part in Bookbuilding. 
The max limit of an investor in book building 500.

This number must a value which we could change easily. 
pls. see if there is this type of validation on the front. 

The error to display in case number is reached. "This Campaign has reached the maximum number of Pre-sale investors. Please consider investing in the Public Sale instead."

**Comment by desfero** [comment]
Agreed with @SergejSoft that we are simply going to hide investor widget if maxim count is reached

**Comment by desfero** [comment]
@falinebambi we need some designs how it should be displayed to the user :)
